### PR TITLE
add polygon tables & update base sources

### PR DESCRIPTION
**Pull request by henrystats**
Brief comments on the purpose of your changes:

@hildobby submitted this [pr](https://github.com/duneanalytics/spellbook/pull/1871) to update the base tables and include polygon tables. 

I copied the updates and submitted it in another pr here  per discussions on discord. 

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributors)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/base_sources/arbitrum_base_sources.yml [hunk 1]
```diff
@@ -10,14 +10,6 @@ sources:
       - name: transactions
         loaded_at_field: block_time
         columns:
-          - name: block_time
```
Comment (jeff-dude) [inline models/base_sources/arbitrum_base_sources.yml:13]: any reason these are being removed? they appear to be in the base table. columns should likely be all or nothing

File: models/base_sources/arbitrum_base_sources.yml [hunk 2]
```diff
@@ -1,41 +1,143 @@
...
 sources:
-# Base Tables
+  # Base Tables
   - name: arbitrum
-    description: "Arbitrum raw tables"
+    description: "Arbitrum raw tables including transactions, traces and logs."
     freshness:
       warn_after: { count: 12, period: hour }
     tables:
+      - name: transactions_0006
```
Comment (jeff-dude) [inline models/base_sources/arbitrum_base_sources.yml:10]: we shouldn't have a need for ```_000*``` tables in spellbook schema files / models

Comment (jeff-dude) [reply models/base_sources/arbitrum_base_sources.yml:10]: any use of transactions will come out of ```transactions``` source

Comment (henrystats) [reply models/base_sources/arbitrum_base_sources.yml:10]: should I update for ethereal as well?

Comment (henrystats) [reply models/base_sources/arbitrum_base_sources.yml:10]: ethereum*

Comment (henrystats) [reply models/base_sources/arbitrum_base_sources.yml]: fixing now.

**Comment by henrystats** [comment]
@jeff-dude removed the 006 and it's breaking.

**Comment by henrystats** [comment]
ready for review ser @jeff-dude

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**